packages/suite-desktop/pages/_app.tsx

```diff
@@ -16,11 +16,42 @@ import DesktopUpdater from '@desktop/support/DesktopUpdater';
 import { SENTRY_CONFIG } from '@suite-config';
 import Resize from '@suite-support/Resize/Container';
 import { isDev } from '@suite-utils/build';
+import styled from 'styled-components';
+import DesktopTitlebar from '@desktop/support/DesktopTitlebar';
 
 interface Props {
     store: Store;
 }
 
+const titlebarHeight = 40;
+const resizeHandlerPadding = 4;
+const TitlebarWrapper = styled.div`
```

Comment:
i'm wondering if it wouldn't be cleaner to have all "component/css" code in one place?
idea:
create `TitlebarWrapper`, `MainWrapper` and `DesktopTitlebar` in one file (support/DesktopTitlebar) , export default as:
```
<TitlebarWrapper>
    <DesktopTitlebar />
</TitlebarWrapper>
<MainWrapper>
    {props.children}
</MainWrapper>
```

and use it here as:

```
<DesktopWrapperOrBetterName>
    <ErrorBoundary>{...and rest of the components...}</ErrorBoundary>
</DesktopWrapperOrBetterName>
```

note that "visual" part should start below `ReduxProvider`

what do you think?

Reply:
should be above or below the ErrorBoundary (to catch potential errors), what do you guys think? @slowbackspace @keraf @mroz22 ?

Reply:
I think the title bar should be outside of the error boundary component otherwise it might get swallowed if an unrelated error happens. The title bar should always be displayed.

Reply:
1) styles - yes, I can split it more
2) constants - agree, maybe I can also add a rule for this to tslint/estlint?
3) regarding ErrorBoundary - I'd keep it as it is - agree with @keraf

Reply:
This part is done in #2663